client/users.go

```diff
@@ -47,22 +47,41 @@ type CreateUserOptions struct {
 	ForceManaged bool   `json:"force-managed,omitempty"`
 }
 
+// RemoveUserOptions holds options for removing a local system user.
+//
+// Username indicates which user to remove.
+type RemoveUserOptions struct {
+	Username string `json:"username,omitempty"`
+}
+
+type userAction struct {
+	Action string `json:"action"`
+	*CreateUserOptions
+	*RemoveUserOptions
+}
+
+func (client *Client) doUserAction(act *userAction) ([]*CreateUserResult, error) {
+	data, err := json.Marshal(act)
+	if err != nil {
+		return nil, err
+	}
+
+	var result []*CreateUserResult
+	_, err = client.doSync("POST", "/v2/users", nil, nil, bytes.NewReader(data), &result)
+	return result, err
+}
+
 // CreateUser creates a local system user. See CreateUserOptions for details.
 func (client *Client) CreateUser(options *CreateUserOptions) (*CreateUserResult, error) {
 	if options.Email == "" {
```

Comment:
Looking at RemoveUser, we should probably update this to do  `options == nil || options.Email == ""`